docs/usage.md

```diff
@@ -91,3 +93,143 @@ Find and set the options which fit to your demands, it is also an option to chan
 | DietPi-RAMlog #1 | RAM | last hour | volatile, i.e. not saved to disk |
 | DietPi-RAMlog #2 | RAM | long term | stored, i.e. hourly saved to disk |
 | Full logging | disk | long term | stored, i.e. immediately saved to disk <br>(with Rsyslog and Logrotate)|
+
+---
+
+## How to do an automatic base installation at first boot (DietPi-Automation)
+
+DietPi offers the option for an automatic first boot installation. Normally, during the first system boot there is an installation procedure which sets up your system initially. The steps described in the section ["First logon on DietPi"](http://localhost:8000/docs/install/#4-first-logon-on-dietpi) are then conducted.
+
+![dietpi-login01](assets/images/dietpi-login01.jpg){: width="640" height="371" loading="lazy"}
+
+These steps need an amount of user interaction which can be overcome with the automatic base installation option described in this section.  
+The automatized setup is based on the configuration file `/boot/dietpi.txt`. It can be edited prior to the first boot and will be evaluated during the first boot procedure. On subsequent boot procedures, the most options in the file are not evaluated any more.
```

Comment:
Probably we should make clear that it is either on the dedicated FAT partition (`dietpi.txt` only then, not in a boot directory) or in the boot directory only when it is a single partition image with an ext4 filesystem only. E.g.
> If the image contains a separate FAT partition, the `dietpi.txt` can be found in its root. If the image contains a single partition only (usually `ext4`), the `dietpi.txt` file can be found within the `boot` directory.

Reply:
I put this clearification into the info box.